### PR TITLE
add iconUrl

### PR DESCRIPTION
**Pull request by adwetzelnr**
# Thanks for submitting a pull request to newrelic/opensource-website.

If possible, please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) standards in your pull request title and commit messages.

## Use `chore`

- for changes / additions / corrections to content in the `src/data/project-main-content`, `src/data/projects` directories
- for changes / additions / corrections to images in the `static/images` directory
- for functional changes / additions to code in the `examples` directory

### Example

```bash
git add src/data/projects/newrelic-nr1-github.json
git add src/data/project-main-content/newrelic-nr1-github.mdx
git commit -m "chore(nr1-github): adjusting config and content"
git push
```

## Use `fix`

- for functional corrections to code in the `src/components`, `src/templates`, `src/fragments`, `src/pages` or directories
- for corrections in the `src/data/oss-category` or `src/data/project-tags` directories
- for functional corrections to code in the `examples` directory

### Example

```bash
git add src/pages/code-of-conduct.mdx
git commit -m "fix: typo and prop error in the code of conduct"
git push
```

## Use `feat`

- for functional changes / additions to code in the `src/components`, `src/templates`, `src/fragments`, `src/pages` or directories
- for changes / additions in the `src/data/oss-category` or `src/data/project-tags` directories

### Example

```bash
git add src/pages/video.js
git commit -m "feat(media): creating a video landing page"
git push
```

### CODE REVIEW

File: src/data/projects/newrelic-video-agent-roku.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "supportUrl": "https://discuss.newrelic.com/t/new-relic-open-source-roku-agent/97802",
   "githubUrl": "https://github.com/newrelic/video-agent-roku",
   "permalink": "https://opensource.newrelic.com/projects/newrelic/video-agent-roku",
-  "iconUrl": null,
+  "iconUrl": "https://github.com/newrelic/video-agent-rokun/blob/master/icon.png?raw=true",
```
Comment (jbeveland27) [inline src/data/projects/newrelic-video-agent-roku.json:13]: Hey @adwetzelnr 👋 Looks like a typo here:
```suggestion
  "iconUrl": "https://github.com/newrelic/video-agent-roku/blob/master/icon.png?raw=true",
```

**Comment by jbeveland27** [comment]
@adwetzelnr Thanks for the contribution. It looks like you haven't signed the CLA yet. If you could do that and update the link, I'll merge this in right away. Thanks!

**Comment by adwetzelnr** [comment]
Signed the CLA, thanks.  And this PR is to fix the typo from my first PR.  So this should supersede the other, if there is such a thing :)

File: src/data/projects/newrelic-video-agent-roku.json (hunk 1 above)
Comment (adwetzelnr) [reply src/data/projects/newrelic-video-agent-roku.json:13]: Thanks for the heads up.  I noticed that too and submitted another PR.  Is there a better way to address/amend a PR that has a problem?

**Comment by tangollama** [comment]
Closing b/c the change was addressed in another PR.

**Pull request closed without merging**